modules/aggregations/build.gradle

```diff
@@ -48,4 +48,5 @@ dependencies {
 
 tasks.named("yamlRestCompatTestTransform").configure({ task ->
   task.skipTest("aggregations/date_agg_per_day_of_week/Date aggregartion per day of week", "week-date behaviour has changed")
+  task.skipTest("aggregations/stats_metric_fail_formatting/fail formatting", "locale has changed")
```

Comment:
do we need a replacement for this test that no longer passes?

Reply:
This is just the BwC test, the same-version test is still running and matches with the locale correctly

Reply:
oh I see, thanks, sounds good.